app/controllers/admin/pet_applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Admin::PetApplicationsController < ApplicationController
+    def update
+        pet_application = PetApplication.where(["application_id = ? and pet_id = ?", "#{params[:id]}", "#{params[:pet_id]}"])
```

Comment:
Since we can be fairly sure there is only one Pet App with the given pet_id and app_id, I think `find_by` would work here and then you wouldn't have to unwrap the record out of an array!

Reply:
Although it looks like the `update` action was smart enough to do the unwrapping for you. But it's good to keep in mind that `where` will return an ARRAY rather than a single record